cmd/server-startup-msg.go

```diff
@@ -76,6 +85,24 @@ func printStartupMessage(apiEndPoints []string) {
 	}
 }
 
+
+//helper message fix - to avoid the docker host ip
+func sanitizeEndpoint(apiEndpoint string) string {
+
+	endpointLen := len(apiEndpoint)
+
+	HOST_ENV_IP := os.Getenv("HOST_IP")
```

Comment:
Thanks for sending the PR.  Now regarding the PR, it would be better to discuss new ENV variables through a GitHub issue so that we can decide and provide feedback @Praveenrajmani - 

Docker printing env is not a problem since docker containers are generally used in a swarm or kubernetes like platforms, on which cloud-native applications directly use the hostname. So this PR I think is not needed as its an overkill.